mpkg/boot2dockerutils.pkg/start.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+b2d="/usr/local/bin/boot2docker"
+unset DYLD_LIBRARY_PATH ; unset LD_LIBRARY_PATH
+mkdir -p ~/.boot2docker
+if [ ! -f ~/.boot2docker/boot2docker.iso ]; then cp /usr/local/share/boot2docker/boot2docker.iso ~/.boot2docker/ ; fi
+$b2d ip
+$b2d init
+$b2d up && export DOCKER_HOST=tcp://$(/usr/local/bin/boot2docker ip 2>/dev/null):2376
```

Comment:
can you change the `export ...` to calling `$(boot2docker shellinit)` - that will automatically set up the entire env, TLS or identity auth, or whatever)